server/src/models/forests.es6

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const url = require('url');
+
+const sequelizeOptions = {
+  dialect: url.parse(process.env.DATABASE_URL, true).protocol.split(':')[0]
+};
+
+if (
+  url.parse(process.env.DATABASE_URL, true).hostname !== 'localhost' &&
+  url.parse(process.env.DATABASE_URL, true).hostname !== 'fs-intake-postgres'
+) {
+  sequelizeOptions.dialectOptions = {
+    ssl: true
+  };
+}
+
+const sequelize = new Sequelize(process.env.DATABASE_URL, sequelizeOptions);
+
+module.exports = sequelize.define(
+  'forests',
```

Comment:
Let's make this forest model, christmas tree specific.

Reply:
We need to this discuss this further

Reply:
For the time being let's create a story @lpendem

Reply:
@lauraGgit story: https://github.com/nciinc/fs-intake-module/issues/79